src/libs/ReportUtils.ts

```diff
@@ -1223,7 +1224,11 @@ function isClosedExpenseReportWithNoExpenses(report: OnyxEntry<Report>): boolean
 /**
  * Whether the provided report is an archived room
  */
-function isArchivedRoom(report: OnyxEntry<Report> | EmptyObject): boolean {
+function isArchivedRoom(report: OnyxEntry<Report> | EmptyObject, reportNameValuePairs?: OnyxEntry<ReportNameValuePairs> | EmptyObject): boolean {
+    if (reportNameValuePairs) {
+        return reportNameValuePairs.isArchived;
+    }
+
     return report?.statusNum === CONST.REPORT.STATUS_NUM.CLOSED && report?.stateNum === CONST.REPORT.STATE_NUM.APPROVED;
```

Comment:
We need to delete this (and the report param) at some point, right?

Reply:
Yes we do, I put some information about that in the "Plan of action" section of his [Design Doc](https://docs.google.com/document/d/1U4vYiyIy0DQ7MVpcsmrDuE_aVNkuLeSqeUgAvn0Sgrc/edit)

Reply:
ok please do not forget (I'd create an issue, but I have no memory)